### PR TITLE
colincogle.name got a little bigger

### PR DESCRIPTION
**Pull request by rhymeswithmogul**
But not by much.  Size increased from 29.9 KB to 36.4 KB.

This PR is:

- [ ] Adding a new domain
- [X] Updating existing domain **size**
- [ ] Changing domain name
- [ ] Removing existing domain from list
- [ ] Website code changes (512kb.club site)
- [ ] Other not listed

- [X] I used the uncompressed size of the site
- [X] I have included a link to the Cloudflare report
- [X] This site is not an ultra minimal site
- [X] The following information is filled identical to the data file

***I confirm that I have read the [FAQ section](https://512kb.club/faq), particularly the two red items around minimal pages and inappropriate content and I attest that this site is neither of these things.***

- [X] Check to confirm

```
- domain: colincogle.name
  url: https://colincogle.name 
  size: 36.4
  last_checked: 2024-06-27
```

Cloudflare Report: https://radar.cloudflare.com/scan/071ec073-a33e-4f3f-9539-c160be2f6828/summary

### CODE REVIEW

File: _data/sites.yml [hunk 1]
```diff
@@ -880,8 +880,8 @@
 
 - domain: colincogle.name
   url: https://colincogle.name/
-  size: 29.9
-  last_checked: 2023-07-07
+  size: 36.4
```
Comment (garritfra) [inline _data/sites.yml:883]:
```suggestion
  size: 52.45
```

**Comment by garritfra** [comment]
Thanks for your honesty. I adjusted the number to be the "Total Bytes", as defined by the FAQ. 🙂

**Comment by rhymeswithmogul** [comment]
Sorry about that! It was late. Each one of these "clubs" has a different testing tool.

**Comment by garritfra** [comment]
All good! That's a good pointer. Might make sense to get that straight.

**Pull request merged**